components/desktop/pulseaudio/WARNING

```diff
@@ -0,0 +1,4 @@
+Attention
```

Comment:
I believe [this](https://github.com/OpenIndiana/oi-userland/pull/15951) is far better than the `WARNING` file.

OTOH, when there is some independent ips repo (http://pkg.toc.de/sunray) containing some independent software (SunRay) then the repo could easily contain any additional packages with additional files that are needed to run the software.

Just my 0.02.

Reply:
Thats the point. For a metapackage we should deliver again the packages gdm, gnome-session and some of their dependencies. 
Regarding Pulseaudio, I'm not sure if SunRay is the only component that requires Pulseaudio client libraries (beyond our IPS repositories).

Reply:
> Thats the point. For a metapackage we should deliver again the packages gdm, gnome-session and some of their dependencies.

If you need some package not delivered by OI, then you can package it and contribute.  If this is not possible (or the package is not acceptable for OI for any reason), then you can add the package to the independent IPS repo.

That does not prevent to have a metapackage with incomplete dependencies.

> Regarding Pulseaudio, I'm not sure if SunRay is the only component that requires Pulseaudio client libraries (beyond our IPS repositories).

I do not think we could (and should) deliver support for any random software found on internet.  If such software is expected to work with OpenIndiana, then it should be packaged for OpenIndiana and contributed to oi-userland.  IOW, anything outside the OI IPS repo cannot reasonably expect to work with OI.  If OpenIndiana would make a commitment to be fully compatible with everything (unspecified) then it would prevent any change in OI, because any change could break the (unspecified) compatibility.